gubernator/github/models.py

```diff
@@ -17,6 +17,7 @@
 import json
 
 import google.appengine.ext.ndb as ndb
+import google.appengine.api.datastore_errors as datastore_errors
```

Comment:
`from google.appengine.api import datastore_errors` is the preferred style: http://google.github.io/styleguide/pyguide.html#224-decision